examples/flows/evaluation/eval-summarization/README.md

```diff
@@ -0,0 +1,106 @@
+
+## Pre-requisites
+
+List any pre-requisites that are required to run the flow. This can include any specific versions of prompt flow or other dependencies. If there are any specific configurations or settings that need to be applied, make sure to mention them in this section. -- TODO remove this when done
+
+Install Prompt Flow SDK and other dependencies in this folder:
+
+```bash
+pip install -r requirements.txt
+```
+
+## Getting Started
+ 
+Provide step-by-step instructions on how to get started with the flow. This should include any necessary setup or configuration steps, such as installing dependencies or setting up connections. If there are specific requirements or prerequisites, make sure to mention them in this section.  -- TODO remove this when done
+
+### Setup connection
+
+Prepare your Azure Open AI resource follow this [instruction](https://learn.microsoft.com/en-us/azure/cognitive-services/openai/how-to/create-resource?pivots=web-portal) and get your `api_key` if you don't have one.
+
+```bash
+# Override keys with --set to avoid yaml file changes
+pf connection create --file ../../../connections/azure_openai.yml --set api_key=<your_api_key> api_base=<your_api_base>
+```
+
+## Usage Examples
+
+Include usage examples that demonstrate how to run the flow and provide input data. This can include command-line instructions or code snippets. Show users how to execute the flow and explain the expected output or results.   -- TODO remove this when done
+
+### 1. Test flow with single line data
+
+```bash
+# test with flow inputs
+pf flow test --flow . --inputs document=ABC summary=ABC
+```
+
+### 2. Create flow run with multi-line data
+
+```bash
+pf run create --flow . --data ./data.jsonl --column-mapping document='${data.document}' summary='${data.summary}' --stream
+```
+
+You can also skip providing `column-mapping` if provided data has same column name as the flow.
+Reference [here](https://aka.ms/pf/column-mapping) for default behavior when `column-mapping` not provided in CLI.
+
+## How to run Unit Tests
+
+1. Make sure you already finished [Prerequisites](#pre-requisites)
+1. Run the following commands
+
+    ```bash
+    pip install pytest
+    python -m pytest tests
+    ```
+
+## Troubleshooting
+
+If there are any known issues or troubleshooting tips related to the flow, include them in this section. Provide solutions or workarounds for common problems that users may encounter. This will help users troubleshoot issues on their own and reduce the need for support.   -- TODO remove this when done, or remove this section if not needed
+
+## Contact
+
+If you have any questions or issues related to this flow, please reach out to either:
+
+- Bhavik Maneck [[Email](mailto:bhavikmaneck@microsoft.com) | [Linked-In](https://www.linkedin.com/in/bhavik-maneck/)]
```

Comment:
do you have a team alias email to put here?

Reply:
we don't have team alias email